custom_components/opensprinkler/strings.json

```diff
@@ -16,6 +16,7 @@
         "data": {
           "url": "URL",
           "password": "Password",
+          "verify_ssl": "Verify SSL",
```

Comment:
I feel like this is a bit vague, since it s kinda a related to security, it should be more explicit like:

- Allow self-signed SSL certificate
- Allow untrusted SSL certificate

For the other languages, if you feel Google translate doesn't cut it we can leave it as EN for now.